tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryResultValueComparator.java

```diff
@@ -261,6 +263,11 @@ private boolean timestampWithTimezoneEqual(Object actual, Object expected)
         return false;
     }
 
+    private boolean javaObjectEqual(Object actual, Object expected)
+    {
+        return actual.equals(expected);
```

Comment:
`java.util.Objects#equals` may be needed

Reply:
turns out not -- https://github.com/prestosql/tempto/blob/100163fb1d99dfde8c4b98fe1753e20d3ce9df56/tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryResultValueComparator.java#L62-L67

Reply:
@findepi Reverted to `actual.equals(expected);` from `java.util.Objects#equals`.